nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -490,24 +515,73 @@ Been a mess since 2018, we'll fix it someday (probably)
 	if((user.client?.prefs.toggles & SOUND_AMBIENCE) && user.can_hear_ambience() && engines_active()) //Disable ambient sounds to shut up the noises.
 		SEND_SOUND(user, sound('nsv13/sound/effects/fighters/cockpit.ogg', repeat = TRUE, wait = 0, volume = 50, channel=CHANNEL_SHIP_ALERT))
 
-/obj/structure/overmap/fighter/stop_piloting(mob/living/M, force=FALSE)
+/obj/structure/overmap/fighter/stop_piloting(mob/living/M, eject_mob=TRUE, force=FALSE)
+	if(eject_mob && !eject(M, force))
+		return FALSE
+	M.stop_sound_channel(CHANNEL_SHIP_ALERT)
+	M.remove_verb(fighter_verbs)
+	return ..()
+
+/obj/structure/overmap/fighter/proc/eject(mob/living/M, force=FALSE)
 	if(!canopy_open && !force)
 		to_chat(M, "<span class='warning'>[src]'s canopy isn't open.</span>")
 		if(prob(50))
 			playsound(src, 'sound/effects/glasshit.ogg', 75, 1)
 			to_chat(M, "<span class='warning'>You bump your head on [src]'s canopy.</span>")
 			visible_message("<span class='warning'>You hear a muffled thud.</span>")
-		return
+		return FALSE
 	if(!SSmapping.level_trait(loc.z, ZTRAIT_BOARDABLE) && !force)
 		to_chat(M, "<span class='warning'>[src] won't let you jump out of it mid flight.</span>")
 		return FALSE
 	mobs_in_ship -= M
-	. = ..()
-	M.stop_sound_channel(CHANNEL_SHIP_ALERT)
 	M.forceMove(get_turf(src))
-	M.remove_verb(fighter_verbs)
 	return TRUE
 
+/obj/structure/overmap/fighter/escapepod/eject(mob/living/M, force=FALSE)
+	. = ..()
+	if(. && !length(mobs_in_ship)) // Last one out means we don't need this anymore
+		qdel(src)
+
+/obj/structure/overmap/fighter/proc/create_escape_pod(path, mob/last_pilot)
+	if(!ispath(path))
+		return
+	// Create pod
+	var/obj/structure/overmap/fighter/escapepod/escape_pod = new path(get_turf(src))
+	if(!istype(escape_pod))
+		message_admins("Unable to create escape pod for [src] with path [path]")
+		return
+	escape_pod.name = "[name] - escape pod"
+	escape_pod.faction = faction
+	escape_pod.desired_angle = 0
+	escape_pod.user_thrust_dir = NORTH
```

Comment:
would be cool if it inherited momentum from it's fighter 😎

Reply:
they're supposed to be really slow though 🤔